spec/services/tag_adjustment_creation_service_spec.rb

```diff
@@ -40,25 +40,29 @@ def create_service(type)
 
   describe "creates notification" do
     it "with adjustment_type removal" do
-      perform_enqueued_jobs do
-        service = create_service("removal")
-        tag_adjustment = service.tag_adjustment
-        tag_adjustment.save
+      service = create_service("removal")
+      tag_adjustment = service.tag_adjustment
+      tag_adjustment.save
+
+      sidekiq_perform_enqueued_jobs do
         service.update_tags_and_notify
-        expect(Notification.last.user_id).to eq(article.user_id)
-        expect(Notification.last.json_data["adjustment_type"]).to eq(tag_adjustment.adjustment_type)
       end
+
+      expect(Notification.last.user_id).to eq(article.user_id)
+      expect(Notification.last.json_data["adjustment_type"]).to eq(tag_adjustment.adjustment_type)
     end
 
     it "with adjustment_type addition" do
-      perform_enqueued_jobs do
-        service = create_service("addition")
-        tag_adjustment = service.tag_adjustment
-        tag_adjustment.save
+      service = create_service("addition")
+      tag_adjustment = service.tag_adjustment
+      tag_adjustment.save
+
+      sidekiq_perform_enqueued_jobs do
         service.update_tags_and_notify
-        expect(Notification.last.user_id).to eq(article.user_id)
-        expect(Notification.last.json_data["adjustment_type"]).to eq(tag_adjustment.adjustment_type)
       end
+
+      expect(Notification.last.user_id).to eq(article.user_id)
+      expect(Notification.last.json_data["adjustment_type"]).to eq(tag_adjustment.adjustment_type)
```

Comment:
Small suggestion, "Notification.last" if we ever get to parallel testing, might not be the same notification you just created, it'd be more future proofing to use `tag_adjustment.reload; tag_adjustment.notifications.last`